### ISSUE TITLE
Add NVDA specific context menu for objects in object navigation mode

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#7795]
Steps to reproduce:
1. Open an application where some buttons or edit fields are not labeled or where the label is not as you want it to be
2. Make that application more accessible to your individual needs (short names for edit fields and so on for more productivity)

Expected behavior:
NVDA should allow me to open a context menu out of the object navigation mode which contains a few options like "show more object information, label object, drag object (if possible; for example drag a text from an edit field into another) etc.". The drag 'n drop function would Ofcourse apply for applications which permit such functions.
NVDA should provide the user with more flexibility to build better accessibility for individual cases. Thus, labels for objects which have been created in a certain application should be saved in a config file which is part of the NVDA version used. The file should be copied to a portable copy as well in case such portable copy is being created.

Actual behavior:
There is no possibility to handle an object in a simple way apart from activating or draging the cursor to it.

System configuration:
NVDA version:
all

NVDA Installed or portable:
Both

Other information:

Windows version:
all

Name and version of other software in use when reproducing the issue:
all

Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
Yes, all

**Comment by fernando-jose-silva** [comment]
I agree with this feature, and it is present in various screen readers for windows and other

**Comment by feerrenrut** [comment]
Thanks for taking the time to raise this, however, this is several feature requests which should normally be split up so that they can be considered independently. Some of the examples you gave already have issues:
* Object labeling already has an issue, see #2111
* Drag and drop, see #3809. Perhaps you have in mind a simpler user interface for drag and drop?
* Could you elaborate on the extra information for an object? Please use a specific example.

I'm going to close this issue, please let me know if I have missed something.

**Comment by feerrenrut** [comment]
I forgot to mention the issue: Drag Mouse Feature #1755 
I have updated the last comment to reflect this.

**Comment by Adriani90** [comment]
Extra information for example coordinates on screen to save position or approximate position on screen (15 cm from left, 10 cm from top) like in powerpoint. Some people would use this to better visualize the screen

**Comment by Adriani90** [comment]
thanks for refered issues. Very interesting.
